docs/usage-exapmle.md

```diff
@@ -36,7 +37,13 @@ fun main(args: Array<String>) {
 ```ts
   const greeting = () => console.log("Hello, world!");
 ```
-<button>Try demo</button>
-<button className="button-secondary">Try demo</button>
-<button className="button-ghost">Try demo</button>
+<button className="button-primary">
+    <img src="../static/img/download-icon.svg" alt=""/>
+    Try demo
+</button> <br/>
+<button className="button-secondary">
+    <Download/>
```

Comment:
Don't import `svg` icons into the page, as they will be indexed as part of the site content. (Use `<Icon>` component and display `svg` via `background-image` instead).